src/components/books.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+
+export default function Books() {
+  const [state, setState] = useState({
+    fname: '',
+    lname: '',
+  });
+
+  const handleChange = (e) => {
+    setState({
+      ...state,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  return (
+    <div>
+
+      <form>
+        <input
+          type="text"
+          name="fname"
+          placeholder="Book title..."
+          value={state.fname}
+          onChange={handleChange}
+        />
+        <input
+          type="text"
+          name="lname"
+          placeholder="Book Author..."
+          value={state.lname}
+          onChange={handleChange}
+        />
+        <button type="button" className="button">Add Book</button>
+      </form>
+      <div className="bookList">jpw</div>
+    </div>
+  );
```

Comment:
- [ ] This component should render two components:

  - It should render a list of books using a **`Book`** component which takes in the `title` and `author` as props. Kindly create a Book component (you do not have one yet) and import it in this component
  - It should also render the Form component. That means you should remove this form feature in this component and import the one you created already in `src/components/form.js`